ipykernel/kernelbase.py

```diff
@@ -855,10 +856,26 @@ async def debug_request(self, stream, ident, parent):
         if inspect.isawaitable(reply_content):
             reply_content = await reply_content
         reply_content = json_clean(reply_content)
+        reply_content['hello'] = 'hello'
         reply_msg = self.session.send(stream, 'debug_reply', reply_content,
                                       parent, ident)
         self.log.debug("%s", reply_msg)
 
+    async def usage_request(self, stream, ident, parent):
+        content = parent['content']
+        reply_content = self.do_debug_request(content)
+        if inspect.isawaitable(reply_content):
+            reply_content = await reply_content
+        reply_content = json_clean(reply_content)
+        reply_content['cpu_percent'] = psutil.cpu_percent()
```

Comment:
Should we ignore 0 values? By doing this we'd keep this part of the API hidden from the front ends who may think 0 is a real value.

https://github.com/ipython/ipykernel/pull/805/files

> Warning the first time this function is called with interval = 0.0 or None it will return a meaningless 0.0 value which you are supposed to ignore.

Reply:
Thx. Fixed with c574a01a344656e0ac915e88dcdc54a9543b92ac